README.md

```diff
@@ -106,6 +106,38 @@ JUDOSCALE = {
 
 Note the [official recommendations for configuring Flask](https://flask.palletsprojects.com/en/2.2.x/config/#configuration-best-practices).
 
+# Using Judoscale with Celery and Redis
+
+> **NOTE 1:** The Judoscale Celery integration currently only works with the [Redis broker](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html#redis).
+
+> **NOTE 2:** Using [task priorities](https://docs.celeryq.dev/en/latest/userguide/calling.html#advanced-options) is currently not supported.
+
+Judoscale can automatically scaling the number of Celery workers based on the queue latency (the age of the oldest pending task in the queue).
+
+To use the Celery integration, import `judoscale_celery` and pass it the Celery "app" instance as the last step, after setup and configuration.
+
+```py
+from celery import Celery
+from judoscale.celery import judoscale_celery
+
+broker = Celery("Broker", broker="redis://localhost:6379/0")
+# Further setup
+# broker.conf.update(...)
+# ...
+
+judoscale_celery(broker)
+```
+
+This sets up Judoscale to periodically calculate and report queue latency for each Celery queue.
+
+Note that the queue names are captured via the [`before_task_publish` signal](https://docs.celeryq.dev/en/latest/userguide/signals.html#before-task-publish) and stored in the metrics collector.
```

Comment:
Is this information relevant to the package consumer? It seems like internal-only details to me.

Reply:
You're right. It's not relevant to the end user.